### PR TITLE
Use rogue instead of pygments.rb which is not Python 3 compatible

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR make `jekyll build` independent of Python version by using rouge instead. See https://github.com/apache/spark/pull/26521 for more details.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
BTW, my local `jekyll build` before this PR makes no difference in the existing HTMLs:

```bash
➜  spark-website git:(asf-site) jekyll build
```
```
Configuration file: /.../spark-website/_config.yml
            Source: /.../spark-website
       Destination: site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 2.896 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
```
```bash
➜  spark-website git:(asf-site) git status
```
```
On branch asf-site
Your branch is up to date with 'origin/asf-site'.

nothing to commit, working tree clean
```

So, this PR includes only the changes after this PR in HTMLs.

**Review by srowen — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged. Thanks @srowen.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ Building the site requires [Jekyll](http://jekyllrb.com/docs) 3.3.0 or newer.
 The easiest way to install jekyll is via a Ruby Gem. This will create a directory called `site` 
 containing `index.html` as well as the rest of the compiled directories and files.
 
-To install Jekyll and its required dependencies, execute `sudo gem install jekyll pygments.rb` 
-and `sudo pip install Pygments`.
+To install Jekyll and its required dependencies, execute `sudo gem install jekyll rogue`.
```
Comment (srowen) [inline README.md:9]: Oops, it's rouge rather than rogue. Others are fixing it in the Docker image. I'll fix this.

Comment (HyukjinKwon) [reply README.md:9]: Argh ..

**Pull request merged**